static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -338,21 +338,35 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
           {
             name: 'githubPRBot',
             type: 'boolean',
-            label: t('Enable Pull Request Bot'),
+            label: t('Enable Comments on Suspect Pull Requests'),
```

Comment:
I don't think this makes it that clear that this is only for closed/merged PRs

Reply:
Maybe change to `Enable comments on Suspect Pull Requests that have been merged or closed`?

Reply:
Maybe that's too long - feel free to give thoughts or reject this

Reply:
this is copy from Dhrumil